### DISCUSSION TITLE
x轴存在滚动条，如何让柱形图不重叠，实现自适应

### DISCUSSION THREAD

**Discussion by Zhousc567** [antvis/G2#5233] [Q&A]
![image](https://github.com/antvis/G2/assets/91332309/8a47d6c0-b143-468d-a4bd-cb9114cfe31a)
![image](https://github.com/antvis/G2/assets/91332309/b9ba8741-7258-4cea-8412-c70c12b93771)
存在滚动条时，设置柱宽度的同时如何让柱形图不重叠。

**Comment by hustcc** [comment]
这个重叠和滚动条没有关系。

**Reply by Zhousc567** [reply]
呢可以实现改变宽度不重叠吗
